WebService/v1/Models/SimulationApiModel/SimulationApiModel.cs

```diff
@@ -103,9 +107,9 @@ public Simulation ToServiceModel(string id = "")
                 Enabled = this.Enabled ?? true,
                 StartTime = DateHelper.ParseDateExpression(this.StartTime, now),
                 EndTime = DateHelper.ParseDateExpression(this.EndTime, now),
-                DeviceModels = this.DeviceModels?.Select(x => x.ToServiceModel()).ToList()
+                DeviceModels = this.DeviceModels?.Select(x => x.ToServiceModel()).ToList(),
+                RateLimits = this.RateLimits?.ToServiceModel()
```

Comment:
note: when merging in master, this will be slightly different - should be similar to how `PartitioningComplete` is managed, e.g. not allowing overwrite after the simulation has been created